backend/routes.js

```diff
@@ -72,6 +72,12 @@ export default class Server {
                     }
                 }, 1000);
             }
+            const awaitCertsAndStartHTTPS = setInterval(() => {
```

Comment:
`await` sounds like the es6 async `await`. could this be renamed to `interval...`?